cucumber-core/src/main/java/io/cucumber/core/options/PickleOrderParser.java

```diff
@@ -1,43 +1,32 @@
 package io.cucumber.core.options;
 
-import io.cucumber.core.logging.Logger;
-import io.cucumber.core.logging.LoggerFactory;
 import io.cucumber.core.order.PickleOrder;
-import io.cucumber.core.order.StandardPickleOrders;
+import io.cucumber.core.order.factory.PickleOrderFactory;
 
-import java.util.Random;
+import java.util.ServiceLoader;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 final class PickleOrderParser {
 
-    private static final Logger log = LoggerFactory.getLogger(PickleOrderParser.class);
-
-    private static final Pattern RANDOM_AND_SEED_PATTERN = Pattern.compile("random(?::(\\d+))?");
+    private static final Pattern NAME_VS_ARGUMENT_PATTERN = Pattern.compile("([a-z_]+)(?::(.+))?");
```

Comment:
I don't think versus is the right word. I would consider `NAME_WITH_ARGUMENT_PATTERN`.